recipes/openms/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# useless default include directory that is silently added by the compiler packages "to help"...
+# it is not even added with -isystem https://github.com/AnacondaRecipes/aggregate/blob/master/clang/activate-clang%2B%2B.sh#L87
+USELESS="-I${PREFIX}/include"
+export CXXFLAGS=${CXXFLAGS//${USELESS}/}
```

Comment:
What's the point of all of this? Can we remove these environment variable defintions?

Reply:
The point is that the compiler packages silently add this to the CXXFLAGS variable.
But this should be taken care of by CMake. Additionally, it adds it with "-I" and not "-isystem" which means that there will be millions of compiler warnings slowing down the build to a point where CI did not finish on bioconda.

Reply:
Maybe this is different for conda-forge though.